app/src/main/java/com/example/development/sakaiclientandroid/NavActivity.java

```diff
@@ -53,28 +55,15 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // Create RequestManager's Retrofit instance
         RequestManager.createRetrofitInstance(this);
-        // Request all site pages for the Home Fragment
-        DataHandler.requestAllSites(new RequestCallback() {
-            @Override
-            public void onCoursesSuccess() {
 
-                spinner.setVisibility(View.GONE);
 
-                Bundle bun = new Bundle();
-                bun.putString("showHomeOrGrades", "Home");
+        //clear the saved tree states in saved preferences so some nodes aren't opened by default
+        SharedPrefsUtil.clearTreeStates(getApplicationContext());
```

Comment:
You can just pass `this` as the context, activities are also contexts. Getting the application context is redundant in this case.